includes/EE_MCI_Controller.class.php

```diff
@@ -238,7 +262,8 @@ public function mci_submit_to_mailchimp($spc_obj, $valid_data)
                         if (! in_array(
                             $att_email,
                             $registered_attendees
-                        ) && (! $need_reg_status || $need_reg_status && $reg_approved)) {
+                        ) && (! $need_reg_status || $need_reg_status && $reg_approved)
```

Comment:
Should this be:
```suggestion
                        ) && (! $need_reg_status || ($need_reg_status && $reg_approved))
```
?